File: cielo24_Java/cielo24/src/cielo24/json/Sequence.java
package cielo24.json;

import java.util.List;
import com.google.gson.annotations.SerializedName;

public class Sequence extends JsonBase {
	@SerializedName("tokens")
	public List<Token> tokens;
	@SerializedName("interpolated")
	public boolean interpolated;
	@SerializedName("start_time")
	public int startTime; // Milliseconds
	@SerializedName("end_time")
	public int endTime; // Milliseconds
	@SerializedName("confidence_score")
	public float confidenceScore;
}

File: cielo24_.NET/Cielo24/JSON/ElementList/Sequence.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using Newtonsoft.Json;

namespace Cielo24.JSON.ElementList
{
    public class Sequence : JsonBase
    {
        [JsonProperty("tokens")]
        public List<Token> Tokens { get; set; }
        [JsonProperty("interpolated")]
        public bool Interpolated { get; set; }
        [JsonProperty("start_time")]
        public int StartTime { get; set; }      // Milliseconds
        [JsonProperty("end_time")]
        public int EndTime { get; set; }        // Milliseconds
        [JsonProperty("confidence_score")]
        public float ConfidenceScore { get; set; }
    }
}